modules/virt-hot-plugging-memory.adoc

```diff
@@ -20,6 +20,10 @@ The system applies these changes immediately. If the VM is migratable, a live mi
 
 [NOTE]
 ====
-* Linux guests require a kernel version of 5.16 or later and Windows guests require the latest `viomem` drivers.
-* By default, you can hot-plug a VM with up to 4 times its initial CPU or memory configuration. For example, a VM with 1 vCPU and 2 GiB memory can scale up to 4 vCPUs and 8 GiB before a restart is needed. You can adjust this behavior cluster-wide by using the `maxHotplugRatio` parameter or override it in your VM.
+*Memory hot-plugging for virtual machines requires guest operating system support for the `virtio-mem` driver. This support depends on the driver being included and enabled within the guest operating system, not on the specific upstream kernel versions.
+Supported guest operating systems:
+* RHEL 9.4 and later
+* RHEL 8.10 and later (hot-unplug is disabled by default)
+* Other Linux guests require kernel version 5.16 or later and the `virtio-mem` kernel module
+* Windows guests require `virtio-mem` driver version 100.95.104.26200 or later
```

Comment:
The formatting needs to be fixed to match the original PR so that it renders correctly:

```suggestion
Memory hot-plugging for virtual machines requires guest operating system support for the `virtio-mem` driver. This support depends on the driver being included and enabled within the guest operating system, not on the specific upstream kernel versions.

Supported guest operating systems:

* RHEL 9.4 and later
* RHEL 8.10 and later (hot-unplug is disabled by default)
* Other Linux guests require kernel version 5.16 or later and the `virtio-mem` kernel module
* Windows guests require `virtio-mem` driver version 100.95.104.26200 or later
```